webpack.config.js

```diff
@@ -208,6 +208,10 @@ if ( calypsoEnv === 'production' ) {
 	) );
 }
 
+if ( ! config.isEnabled( 'desktop' ) ) {
```

Comment:
We are exposing object from `lib/desktop` and use different methods from it, so I don't think it would work this way.